### ISSUE TITLE
BUG: run_openai_gpt.py load ROCStories data error

### ISSUE THREAD

**Issue by nine09** [huggingface/pytorch-transformers#1051]
## 🐛 Bug

Model I am using (Bert, XLNet....): GPT

Language I am using the model on (English, Chinese....): English

The problem arise when using:
* [ 1 ] the official example scripts: run_openai_gpt.py

The tasks I am working on is:
* [ 1 ] an official GLUE/SQUaD task: ROCStories

**The method of preprocess of ROCStories use ids 0 pad input, while 0 is actually the id of unk_tokens.**

The code of method is as fellowing:
```
def pre_process_datasets(encoded_datasets, input_len, cap_length, start_token, delimiter_token, clf_token):
    tensor_datasets = []
    for dataset in encoded_datasets:
        n_batch = len(dataset)
        input_ids = np.zeros((n_batch, 2, input_len), dtype=np.int64)
        for i, (story, cont1, cont2, mc_label), in enumerate(dataset):
            with_cont1 = [start_token] + story[:cap_length] + [delimiter_token] + cont1[:cap_length] + [clf_token]
            with_cont2 = [start_token] + story[:cap_length] + [delimiter_token] + cont2[:cap_length] + [clf_token]
            input_ids[i, 0, :len(with_cont1)] = with_cont1
            input_ids[i, 1, :len(with_cont2)] = with_cont2
```
input_ids initialize with 0, which is the id of unk_token, rather than id of pad_token.

**Comment by julien-c** [comment]
Hi @nine09, thanks for the report. Any way you could fix it cleanly and open a pull request?

**Comment by nine09** [comment]
> Hi @nine09, thanks for the report. Any way you could fix it cleanly and open a pull request?

Bien sur! But I need some clues about whether GPTTokenizer already contains some special tokens and how to add special tokens to GPTTokenizer and resize the embedding size of GPTModel.

**Comment by nine09** [comment]
> Hi @nine09, thanks for the report. Any way you could fix it cleanly and open a pull request?

Bien sur! But I want to have some clues about whether GPTTokenizer already have pad_token, otherwise add a new pad_token need resize embedding of GPTModel.

**Comment by nine09** [comment]
I have fix the bug at #1064, that change add a pad_token to GPTTokenizer so that solved this problem.
